magnetic/Glk/glk.c

```diff
@@ -1827,6 +1827,29 @@ gms_graphics_paint_region (winid_t glk_window, glui32 palette[], int layers[],
     }
 }
 
+static void
+gms_graphics_paint_everything (winid_t glk_window,
```

Comment:
Looks like this is only used by Gargoyle?  If so, it should be protected with `#ifdef GARGLK`

Reply:
This one confuses me. The function is used only if Garglk fail to paint a picture, but it defined in [the glk.c code](https://github.com/cspiegel/interpreters/blob/70bdb286b98350b9fe20bacc42de0c963070cc16/magnetic/Glk/glk.c#L1830,L1852). In theory it could be called anywhere.